modules/documents/client/directives/documents.manager.directive.js

```diff
@@ -6,23 +6,26 @@ angular.module('documents')
 			restrict: 'E',
 			scope: {
 				project: '=',
-				opendir: '='
+				file: '=',
+				folder: '='
 			},
 			templateUrl: 'modules/documents/client/views/document-manager.html',
 			controller: function ($scope, $filter, $log, $modal, $timeout, _, moment, Authentication, DocumentMgrService, CodeLists, TreeModel, ProjectModel, Document) {
 				var tree = new TreeModel();
 				var self = this;
 				self.busy = true;
+				self.requestOpenDir = null;
+				self.requestOpenFileID = null;
 
-				if ($scope.opendir) {
+				if ($scope.file) {
+					self.requestOpenFileID = $scope.file; // objectId
+				}
+				if ($scope.folder) {
 					try {
-						self.opendir = $scope.opendir.substr(1,$scope.opendir.length - 1);
-						self.opendir = self.opendir.split('=');
-						self.opendir = parseInt(self.opendir[1]);
+						self.requestOpenDir = parseInt($scope.folder); // tree Id
```

Comment:
The parseInt is in try catch. Logic below tested requestOpenDir for truthyness but I'll modify to test for NaN